src/Command/GetClientCommand.php

```diff
@@ -0,0 +1,41 @@
+<?php
+namespace App\Command;
+
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\HttpClient\HttpClient;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputOption;
+
+class GetClientCommand extends BaseCommand
+{
+    protected function configure(): void
+    {
+        parent::configure();
+        $this
+            ->setName('client:details')
+            ->setDescription('Get a client')
+            ->addArgument('id', InputArgument::REQUIRED, "Client's id")
+            ->addOption('output', 'o', InputOption::VALUE_REQUIRED, "Output file to save client")
+        ;
+    }
+    
+    protected function execute(InputInterface $input, OutputInterface $output)
+    {
+        $httpClient = HttpClient::create();
+        $url = $input->getOption('apiUrl');
+        $username = $input->getArgument('username');
+        $password = $input->getArgument('password');
+        $id = $this->parseInt($input->getArgument('id'));
```

Comment:
This will throw an uncontrolled exception if it is not an int.
This is a probable mistake by the user, not an exception, we should catch it, output an error message and return a suitable error value.